lib/livebook_web/live/hub/secret_list_component.ex

```diff
@@ -95,7 +95,7 @@ defmodule LivebookWeb.Hub.SecretListComponent do
         :ok ->
           socket
           |> put_flash(:success, "Secret #{secret.name} deleted successfully")
-          |> push_navigate(to: ~p"/hub/#{hub.id}")
+          |> push_patch(to: ~p"/hub/#{hub.id}")
```

Comment:
Actually, this isn't allowing the LiveView to update the secrets list, but it works when adding/updating the secret.

Reply:
Ah, it's because the path doesn't change, for adding/updating the form is at a different URL that we can patch back from, but for delete we use the path-less modal. One thing we could do is send a message to self instead to reload the hub.

Reply:
I implemented back the `counter` and now it works

Reply:
FTR the issue is specific to delete, because we pass `live_action={@live_action}` to the components, for add and edit this assign changes and forces re-render (where we fetch secrets). For delete however, all assigns are unchanged and the component update is never called.

One alternative would be to explicitly send update event to the component, whenever we want to re-fetch secrets, but doing that in all cases and for file systems, deployment groups is more work, so I think the counter is fine in this case :)